src/Waives.Http/WaivesClient.cs

```diff
@@ -54,6 +59,16 @@ public int Timeout
             set => HttpClient.Timeout = TimeSpan.FromSeconds(value);
         }
 
+        internal ILogger Logger
```

Comment:
✋ This seems to be repeating the problems we have with the `HttpClient` member, where the responsibility is implemented with both the lower-level abstraction and the higher-level abstraction in the same class. Is this change required for the PR, or can it be undone and tackled separately later?

Reply:
I'll tidy it up properly in the next PR. It fixes an issue (albeit untidily) with log messages not being sent to the specified logger when using the pipeline.